Tests/SchebTwoFactorBundleTest.php

```diff
@@ -27,7 +27,7 @@ class ContainerBuilderMock extends ContainerBuilder
 {
     public $passes = array();
 
-    public function addCompilerPass(CompilerPassInterface $pass, $type = PassConfig::TYPE_BEFORE_OPTIMIZATION, $priority = 0)
+    public function addCompilerPass(CompilerPassInterface $pass, $type = PassConfig::TYPE_BEFORE_OPTIMIZATION, int $priority = 0)
```

Comment:
Would you please remove the int type hint? The bundle officially still supports PHP5, so let's keep it to that language level.